### PR TITLE
Adds Scala Exercises to list of resources for learners

### PR DESCRIPTION
**Pull request by raulraja**
@non @adelbertc This PR adds Scala Exercises Cats section to the list of resources for learners.

### CODE REVIEW

File: docs/src/site/resources_for_learners.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ section: "resources_for_learners"
 
 # Tutorials
  * [herding cats](http://eed3si9n.com/herding-cats/) by Eugene Yokota
+ * [Scala Exercises](https://www.scala-exercises.org/cats) by 47 Degrees 
+   offers multiple online exercises to learn about the typeclasses and
```
Comment (fthomas) [inline docs/src/site/resources_for_learners.md]: @raulraja Could you please change "typeclasses" to "type classes". We agreed on this spelling in #441. :-)

Comment (raulraja) [inline docs/src/site/resources_for_learners.md]: @fthomas done!, thanks.

**Comment by fthomas** [comment]
:+1:

**Comment by adelbertc** [comment]
:+1:

**Pull request merged**